src/Raven.Client/Server/Operations/ConfigureExpirationBundleOperation.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Server.expiration;
+using Sparrow.Json;
+
+namespace Raven.Client.Server.Operations
+{
+    public class ConfigureExpirationBundleOperation : IServerOperation<ConfigureExpirationBundleOperationResult>
```

Comment:
Misunderstood. I think you are talking about my wish to break our json.net dependency entirely?